sdk/websites/Azure.ResourceManager.AppService/src/Customization/HybridConnectionData.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.ResourceManager.AppService
+{
+    public partial class HybridConnectionData
+    {
+        /// <summary> The ARM URI to the Service Bus relay. </summary>
```

Comment:
Consider provide more info for developer to use the new property.

Reply:
I added some information in the message of Obsolete attribute